enterprise/models.py

```diff
@@ -2325,9 +2325,12 @@ class EnterpriseCustomerReportingConfiguration(TimeStampedModel):
         (DATA_TYPE_GRADE, DATA_TYPE_GRADE),
         (DATA_TYPE_COMPLETION, DATA_TYPE_COMPLETION),
         (DATA_TYPE_COURSE_STRUCTURE, DATA_TYPE_COURSE_STRUCTURE),
-
     )
 
+    # This list will specifies the Data Types that are allowed to send without compression.
+    # All other Data Types must be compressed.
```

Comment:
Nit:
```python
# This list will specifies the Data Types that are allowed to send without compression.    
# All other Data Types must be compressed.
-> 
# Data types that are allowed to be sent without compression, all other data types must be compressed.

Reply:
Also, shouldn't we handle the `invalid-name` warning?

Reply:
@sameenfatima78 actually its name is too long. the valid name should be less than 30 chars.

I am unable to think of any name that is less than 30 chars any, meaningful.
you can suggest any if you have any.